server/controllers/verify.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require('jsonwebtoken');
+require('env2')('./config.env');
+console.log('fffffffffffffffffffffffff');
+
+const verifyCookie = (req, res, next) => {
+  if (req.cookies.accessToken === undefined) {
+    console.log('notttttt allooe');
+    
+    // res.redirect('/');
```

Comment:
Here, you could return send a 401 (unauthorized) response.  That'll make things pretty clear in the frontend code what happened (on apps I've worked on, we then handle the 401 response by redirecting the user to the login page).